EasyPost.Tests.Net/BatchTest.cs

```diff
@@ -1,232 +1,172 @@
-                    }
-                },
-                {
-                    "to_address", toAddress
-                },
-                {
-                    "from_address", fromAddress
-                },
-                {
-                    "carrier", "USPS"
-                },
-                {
-                    "service", "Priority"
-                }
-            };
+            });
         }
 
         [TestMethod]
-        public void TestAddRemoveShipments()
+        public void TestCreate()
         {
-            Batch batch = Batch.Create();
-            Shipment shipment = Shipment.Create(shipmentParameters);
-            Shipment otherShipment = Shipment.Create(shipmentParameters);
+            VCR.Replay("create");
 
-            while (batch.state != "created")
-            {
-                batch = Batch.Retrieve(batch.id);
-            }
+            Batch batch = CreateBasicBatch();
 
-            batch.AddShipments(new List<Shipment>
+            Assert.IsInstanceOfType(batch, typeof(Batch));
+            Assert.IsTrue(batch.id.StartsWith("batch_"));
+            Assert.IsNotNull(batch.shipments);
+        }
+
+        [TestMethod]
+        public void TestRetrieve()
+        {
+            VCR.Replay("retrieve");
+
+            // use "testCreate"
+            Batch batch = CreateBasicBatch();
+
+            Batch retrievedBatch = Batch.Retrieve(batch.id);
+
+            Assert.IsInstanceOfType(retrievedBatch, typeof(Batch));
+            Assert.AreEqual(batch.id, retrievedBatch.id);
+        }
+
+        [TestMethod]
+        public void TestAll()
+        {
+            VCR.Replay("all");
+
+            BatchCollection batchCollection = Batch.All(new Dictionary<string, object>
             {
-                shipment,
-                otherShipment
+                {
+                    "page_size", Fixture.PageSize
+                }
             });
 
-            while (batch.shipments == null)
+            List<Batch> batches = batchCollection.batches;
+
+            Assert.IsTrue(batches.Count <= Fixture.PageSize);
+            Assert.IsNotNull(batchCollection.has_more);
+            foreach (var item in batches)
             {
-                batch = Batch.Retrieve(batch.id);
+                Assert.IsInstanceOfType(item, typeof(Batch));
             }
+        }
 
-            List<string> shipmentIds = batch.shipments.Select(ship => ship.id).ToList();
-            Assert.AreEqual(batch.num_shipments, 2);
-            CollectionAssert.Contains(shipmentIds, shipment.id);
-            CollectionAssert.Contains(shipmentIds, otherShipment.id);
-
-            batch.RemoveShipments(new List<Shipment>
-            {
-                shipment,
-                otherShipment
-            });
-            Assert.AreEqual(batch.num_shipments, 0);
+        [TestMethod]
+        public void TestCreateAndBuy()
+        {
+            // TODO: C# doesn't have a CreateAndBuy method for Batch
```

Comment:
Is there a way we can move this comment to the `skip` section for the test so it gets called out when run?

Reply:
Yes, I'll mark it as ignore. Planning to fix this later this week in a separate PR.

Reply:
Resolved in 88726861fba93fd348a76b99a851ecbdaf1c49d5